config.py

```diff
@@ -5,6 +5,8 @@
 srvname = 'kdlp.underground.software'
 production = False
 
+use_podman = False
```

Comment:
I think doing this via config is overkill, just allow overriding in invocation of hyperspace by setting an env var, user can make an alias if they want

Reply:
I was thinking we could use this for some other small utilities but I can always add it in later